tests/io_timeout.rs

```diff
@@ -3,6 +3,7 @@ use std::time::Duration;
 use async_std::io;
 use async_std::task;
 
+#[ignore]
```

Comment:
https://github.com/async-rs/async-std/pull/286#issuecomment-543616225
> This PR should not have changed the behavior on macOS, but [this test](https://github.com/async-rs/async-std/blob/a14b11c0325cd362295d9558e2028f51ca3ff5f2/tests/io_timeout.rs#L6-L19) failed on macOS on CI... But I couldn't reproduce this locally.

...but, since it is a comment more than 2 weeks ago, I will delete that `#[ignore]` and see if CI succeeds.